src/js/Controller/Controller.js

```diff
@@ -0,0 +1,120 @@
+
+import { APIKEY } from "../../../config.js"
+import { data } from "../fakeObject.js"
+import { STORAGETYPE } from "../util/Static.js"
+import { qsAll } from "../helpers.js"
+import toWatchView from "../View/ToWatchView.js"
+import watchedView from "../View/WatchedView.js"
+import likedView from "../View/LikedView.js"
+
+const tag = "[Controller]"
+
+
+export default class Controller {
+
+  constructor({ modalView, modalSearchResult, modalScrollView, modalLatestKeywordList, checkBtnView }) {
+
+    this.modalView = modalView
+    this.modalSearchResult = modalSearchResult
+    this.modalScrollView = modalScrollView
+    this.modalLatestKeywordList = modalLatestKeywordList
+    this.checkBtnView = checkBtnView
+
+    this.nextPageToken = ""
+    this.beforeInputValue = ""
+    this.subscribeViewEvents()
+    this.render()
+  }
+  subscribeViewEvents() {
+    this.modalView.on("@submit", event => this.search(event.detail.value))
+    this.modalView.on("@save", event => this.saveVideo(event.detail.value))
+    this.modalScrollView.on("@scroll", event => this.search(event.detail.value))
+    this.checkBtnView.on("@watch", event => this.checkBtnClick(event.detail.value))
+    this.checkBtnView.on("@del", event => this.delBtnClick(event.detail.value))
+    this.checkBtnView.on("@like", event => this.likeBtnClick(event.detail.value))
```

Comment:
```suggestion
    const { modalView, modalScrollView, checkBtnView } = this
    modalView.on("@submit", ({ detail }) => this.search(detail.value))
    modalView.on("@save", ({ detail }) => this.saveVideo(detail.value))
    modalScrollView.on("@scroll", ({ detail }) => this.search(detail.value))
    checkBtnView.on("@watch", ({ detail }) => this.checkBtnClick(detail.value))
    checkBtnView.on("@del", ({ detail }) => this.delBtnClick(detail.value))
    checkBtnView.on("@like", ({ detail }) => this.likeBtnClick(detail.value))
```
이렇게 중복코드를 제거해주면 어떨까요?